File: ARCADE/Intro/Level 10_1: isBeautifulString.cs
    using System;
     
    public class Test
    {
    	public static bool isBeautifulString(string inputString){
            bool result = true;
            for (char now = 'a'; now < 'z'; now++){
                char next = (char)(now + 1);
                if (Count(inputString, now) < Count(inputString, next)){
                    result = false;
                    break;
                }
            }
            return result;
        }
        public static int Count(string s, char arr){
            int count = 0;
            for (int i = 0; i < s.Length; i++)
            {
                if (s[i] == arr) count++;
            }
            return count;
        }
    	public static void Main()
    	{
    		string inputString = "bbbaacdafe";
    		Console.WriteLine(isBeautifulString(inputString));
    		string inputString1 = "aabbb";
    		Console.WriteLine(isBeautifulString(inputString1));
    	}
    }

File: ARCADE/Intro/Level 10_1: isBeautifulString.pas
program isBeautifulString;
 
function countChar(s: string; ar: char): integer;
var i, count: integer;
begin
    count := 0;
    for i := 1 to length(s) do
        if(s[i] = ar) then inc(count);
    countChar := count;
end;
 
 
function isBeautifulString(inputString: string): boolean;
var ar, ar1 : char;
    rs : boolean;
begin
    rs := true;
    for ar := 'a' to 'y' do
    begin
        ar1 := chr(ord(ar) + 1);
        if(countChar(inputString, ar) < countChar(inputString, ar1)) then
        begin
            rs := false;
            break;
        end;
    end;
    isBeautifulString := rs;
end;
 
var inputString : string;
begin
	inputString := 'bbbaacdafe';
	writeln(isBeautifulString(inputString));
	inputString := 'aabbb';
	writeln(isBeautifulString(inputString));
	inputString := 'bbc';
	writeln(isBeautifulString(inputString));
end.
